public/docs/ts/latest/guide/user-input.jade

```diff
@@ -150,76 +148,67 @@ figure.image-display
 :marked
   ## On blur
 
-  Our previous example won't transfer the current state of the input box if the user mouses away and clicks
-  elsewhere on the page. We update the component's `values` property only when the user presses Enter
+  If the user mouses away and clicks elsewhere on the page,
+  the previous example won't transfer the current state of the input box.
+  You update the component's `values` property only when the user presses Enter
   while the focus is inside the input box.
 
-  Let's fix that by listening to the input box's blur event as well.
+  To fix that, listen to the input box's blur event as well.
 
 +makeExample('user-input/ts/app/keyup.components.ts', 'key-up-component-4' ,'app/keyup.components.ts (v4)')(format=".")
 
 .l-main-section
 :marked
   ## Put it all together
-  We learned how to [display data](./displaying-data.html) in the previous chapter.
-  We've acquired a small arsenal of event binding techniques in this chapter.
+  The previous chapter showed how to [display data](./displaying-data.html).
```

Comment:
@wardbell We started that when we switched to the new style guide. I don't remember the history of the decision (Naomi might), but it's in the checklist. I should add it to the word list.